### PR TITLE
release(v0.3.0-alpha.1): prepare release

### PR DESCRIPTION
**Pull request by andrewrynhard**
This is the official v0.3.0-alpha.1 release.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,244 @@
...
+*This is a pre-release of Sidero*
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/sidero/issues.
+
+### ServerClass `any`
+
+Sidero now creates ServerClass `any` which matches all servers.
```
Comment (AlekSi) [inline CHANGELOG.md:13]: Should we also mention `default` environment?

Comment (smira) [reply CHANGELOG.md:13]: Yes, we should, in general we should add to hack/release.toml as we introduce new features. As we missed that, we should do this as part of this PR

**Pull request merged**